src/fu-install-task.c

```diff
@@ -418,6 +418,15 @@ fu_install_task_check_requirements (FuInstallTask *self,
 			return FALSE;
 		}
 	}
+#ifndef HAVE_POLKIT
+	if ((self->trust_flags & FWUPD_TRUST_FLAG_PAYLOAD) == 0) {
+		g_set_error_literal (error,
+				     FWUPD_ERROR,
+				     FWUPD_ERROR_INVALID_FILE,
+				     "untrusted release");
```

Comment:
maybe something more helpful, like `archive has not been signed and release is untrusted`?

Reply:
Well you don't know that it's not signed, it might be signed but with the wrong signature.  So maybe `archive signature not trusted`?

Reply:
I also think this is in the wrong "layer", i.e. the logic is sound but it probably should live in `fu_engine_check_requirements` instead.

Reply:
> archive signature not trusted

maybe `archive signature missing or not trusted`?